src/server.js

```diff
@@ -202,6 +203,88 @@ app.get(
   }
 );
 
+function averageGeolocation(coords) {
+  if (coords.length === 1) {
+    return coords[0];
+  }
+  let x = 0.0;
+  let y = 0.0;
+  let z = 0.0;
+
+  for (let coord of coords) {
+    let latitude = (coord.latitude * Math.PI) / 180;
+    let longitude = (coord.longitude * Math.PI) / 180;
+
+    x += Math.cos(latitude) * Math.cos(longitude);
+    y += Math.cos(latitude) * Math.sin(longitude);
+    z += Math.sin(latitude);
+  }
+
+  let total = coords.length;
+
+  x = x / total;
+  y = y / total;
+  z = z / total;
+
+  let centralLongitude = Math.atan2(y, x);
+  let centralSquareRoot = Math.sqrt(x * x + y * y);
+  let centralLatitude = Math.atan2(z, centralSquareRoot);
+
+  return {
+    latitude: (centralLatitude * 180) / Math.PI,
+    longitude: (centralLongitude * 180) / Math.PI,
+  };
+}
+
+let sessionName = "Get together";
+
+// Mock data; expect ~ 37.790831, -122.407169
+const sf = [
+  {
+    latitude: 37.797749,
+    longitude: -122.412147,
+  },
+  {
+    latitude: 37.789068,
+    longitude: -122.390604,
+  },
+  {
+    latitude: 37.785269,
+    longitude: -122.421975,
+  },
+];
+
+// Mock data; expect ~ 8.670552, -173.207864
+const globe = [
```

Comment:
is this ever used?

Reply:
It's never called. I put it in there for testing how things will work when the co-ordinates are far apart. It may not be necessary as the mock data will be taken off and there may be a feature limiting how far apart the users are